src/ProgramVariable.cpp

```diff
@@ -74,5 +84,32 @@ std::set<std::string> ProgramVariable::getAllAliases(bool cleanNames) {
       allAliases.insert(pv.getRawName());
     }
   }
+  return allAliases;
+}
+
+std::list<ProgramVariable *> ProgramVariable::getNamedAliasesVars() {
```

Comment:
The difference between `getNamedAliases, getAllAliases` and `getNamedAliasesVars, getAllAliasesVars` is just in the return type.  `getNamedAliases, getAllAliases` returns a set of strings representing its aliases, and `getNamedAliasesVars, getAllAliasesVars` returns a list of the original `ProgramVariable*`s representing the aliases; I added these methods because it could be useful to retrieve the original LLVM `Value*` stored inside a `ProgramVariable`, but I just realized I never actually used these new methods, so I might end up removing them. Also, could I add the comments to `ProgramVariable.h` instead?

Reply:
Adding the comment to `ProgramVariable.h` instead is fine.

For methods that aren't being used, I agree that it would be better to remove them.